packages/altive_lints/lib/src/lints/avoid_hardcoded_color.dart

```diff
@@ -45,11 +45,10 @@ class AvoidHardcodedColor extends DartLintRule {
     CustomLintContext context,
   ) {
     context.registry.addInstanceCreationExpression((node) {
-      final typeName =
-          node.staticType?.getDisplayString(withNullability: false);
+      final typeName = node.staticType?.getDisplayString();
 
       if (typeName == 'Color') {
-        reporter.reportErrorForNode(code, node);
+        reporter.atNode(node, code);
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important) 
It seems that an error is occurring with the analyze here!  
It might be better to pass `_lintCode`. Additionally, it would be helpful if you could change `_lintCode` to `_code` to match the other rules!

Reply:
It seems to occur with analyzer v6.8.0 or higher.
Thanks!I'll make sure to use it after _lintCode Rename👍